### DISCUSSION TITLE
Use Marp to build a resume ? or A4 Document ?

### DISCUSSION THREAD

**Discussion by amineHY** [marp-team/marp#370] [Ideas] [answered]
**Marp** is a powerful tool that exploits Markdown files and turns them into slides. 

is there a possibility to extend Marp to build an A4 document possible? like a resume? or a simple A4 PDF document

instead of `size: 16:9` use `size: A4` for example

What do you think?

**Comment by yhatt** [comment, marked as answer]
Yes, you can create [the custom theme CSS](https://marpit.marp.app/theme-css) according to your purpose. The custom slide size also can define in the theme: https://marpit.marp.app/theme-css?id=slide-size

- https://github.com/marp-team/marp-vscode/issues/43
- https://github.com/marp-team/marpit/issues/248
- https://github.com/marp-team/marp/discussions/356

And https://twitter.com/bpedigod/status/1514337012874194955 is an interesting one too.

**Comment by Octobug** [comment]
I've made a demo repo: <https://github.com/Octobug/marp-vscode-resume>
